build_tools/buildkite/scripts/wait_for_pipeline_success.py

```diff
@@ -232,6 +65,10 @@ def parse_args():
       " on build finished in state other than 'passed'",
       choices=["force", "failed", "bad"],
   )
+  parser.add_argument(
+      "--output-build-json",
+      help="Output path of the build object in JSON, which contains the details"
```

Comment:
Should we output the build json regardless of whether it passed?

Reply:
I put a default value to this option.

Reply:
I don't think a default path makes sense. I meant why don't we output the build json regardless of whether or not the build passed (see suggestions)